Tasks/Misiuk/Task1/Program4_converter.py

```diff
@@ -0,0 +1,6 @@
+print('Введите колличество USD которые хотите перевести в BYN.')
+print('Курс на сегодня: 1 USD = 2.85 BYN.')
+USD = int(input("Введите сумму: "))
+BYN = 2.85
+print('--------------ИТОГ------------------')
+print(f'За {USD}$, вы получите {USD * BYN:.2f} BYN')
```

Comment:
correct rounding, but I advise you to use decimal for money-related calculations